client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-identify/index.tsx

```diff
@@ -96,7 +96,10 @@ const SiteMigrationIdentify: Step = function ( { navigation } ) {
 	);
 
 	const urlQueryParams = useQuery();
-	const isEntrepreneurSignup = urlQueryParams.get( 'ref' ) === 'entrepreneur-signup';
+	const shouldHideBack = () => {
```

Comment:
Not a blocker!

We have a bunch of `is<whatever>Flow` function in [flows.ts](https://github.com/Automattic/wp-calypso/blob/trunk/packages/onboarding/src/utils/flows.ts).

Would it be worth creating a new function for this flow?

Reply:
I am not sure what's the story behind `entrepreneur-signup`. It's not listed on the constants in [flows.ts](https://github.com/Automattic/wp-calypso/blob/trunk/packages/onboarding/src/utils/flows.ts).
I think it's better to leave it this way to prevent opening a can of worms.

Reply:
Makes sense to me. I wondered about `entrepreneur-signup` too.